### ISSUE TITLE
Line numbers don't line up in gutter with ProggyTiny font

### ISSUE THREAD

**Issue by The-Compiler** [atom/atom#9344]
I'm trying to use the [ProggyTiny](http://www.proggyfonts.net/download/) font with Atom.

It's a bitmap font, i.e. designed to be viewed at a certain size. When I set the size to `16` (the only size where it looks right - I usually need to use `12` with other applications), the line spacing is much too big:

![atom_1](https://cloud.githubusercontent.com/assets/625793/10842484/33e71248-7ef2-11e5-9c13-038a162f77c5.png)

When setting the line spacing to `1.0` instead of the default `1.5`, the line numbers start to act funny:

![atom_2](https://cloud.githubusercontent.com/assets/625793/10842500/640fe56c-7ef2-11e5-9f52-33fb7c270bba.png)

It's even worse with `0.8`:

![atom_3 png](https://cloud.githubusercontent.com/assets/625793/10842520/87256fe0-7ef2-11e5-8ca8-2f8529045203.png)

This is with Atom 1.0.19 on Debian Jessie, and I see this with the default themes (tried "Atom Dark" and "One Dark").

**Comment by damieng** [comment]
Hi, can you try this:

1. Open **File **> **Settings** from the menu
2. Click **Open Config Folder **
3. Open **styles.less** from the left pane
4. Modify the atom-text-editor block so it shows:

```css
atom-text-editor {
   font-size: 12pt;
}
```

And let me know if that looks any better.

**Comment by The-Compiler** [comment]
I'm unfortunately not using Atom anymore, so I'd be glad if someone else could try to reproduce it. Otherwise I'll try myself, but it might take me some days.

**Comment by The-Compiler** [comment]
I just discovered I still had Atom installed at work, so I could give this a quick try - unfortunately that doesn't help, it looks the same.

Also I'm not sure why this has the "windows" label seeing that I have this on Debian Jessie :wink:

**Comment by Edenharder** [comment]
A similar bug appears when I set the font-size of line number to smaller number, for example 12px. Pls look at the screenshot of this bug.
<img width="498" alt="screen shot 2016-10-13 at 8 57 06 am" src="https://cloud.githubusercontent.com/assets/6672134/19339520/2a87ae34-9124-11e6-91c2-baca65356df9.png">

**Comment by CharlSteynberg** [comment]
Why is this issue closed if this exact problem persists? ... closing the issue does not resolve the issue .. .. i mean like .. really? ..amazing.. it's like a fault is reported in a nuclear reactor; to resolve the issue - the issue is trashed .. bravo to the automatic genius - get rid of the issue - no issue! :dancer:

**Comment by CharlSteynberg** [comment]
For **ProggyTiny** fonts, the following **[SOLVED]** the issue:

```css
.editor
{
    font-family: 'ProggyTinyTT';
    font-size: 16px;
    line-height: 17px;
}

```
In English:  because ProggyTiny fonts look best @ 16px, setting the line-height to font-size + 1px solves the issue (kind-of) .. if you're not too fussed about exact line-height -it's fine. .. must be a percentage (em?) scaling thing in the gutter, not sure, assumption is.. dangerous.. anyway, I hope it helps someone.

**Comment by skylize** [comment]
@CharlSteynberg, if you read the message from the bot that closed the issue, it tells you exactly why it was closed. Since it is still reproducable, I would expect it to get re-opened.

As for work-around. I use line-number padding to correct for fonts that don't line up.
```
atom-text-editor {
  .line-number{
    padding-top: 0.14em;
    padding-bottom: 0.14em;
  }
}
```

**Comment by skylize** [comment]
@damieng Hmm... I was going to argue with the claim that it was only on one font, because I deal with a similar issue.

But with a little testing before posting, I remembered that my misaligned line-numbers were caused by changing the font-size of the scroll-view (so that <kbd>Ctrl</kbd>-<kbd>0</kbd> would zoom to my preferred font-size instead of default 14px).

```
// causes line-number mis-alignment for any font
`atom-text-editor .scroll-view {
  font-size: 1.21em;
}
```

So at the moment, _I_ am unable to reproduce OP's case with another font.
